content/flare/consensus.md

```diff
@@ -287,3 +287,223 @@ Market forces cause quorum slices to overlap and this gives rise to the network-
+
+Aleph protocol is an example of a Total Ordering algorithm that creates a total order of blocks, thus creating a finalized chain at run, and keep its DAG history records only locally.
+This gives to the nodes a higher level of independence when executing the algorithm, and the weight of synchronizing towards reaching the same total order of blocks is transferred to the combinatoric selection of messages from the Byzantine flow.
+
+### Blockmania
+
+Blockmania is also a leaderless Byzantine Fault Tolerant consensus protocol, detailed in the [Blockmania academic paper](https://arxiv.org/abs/1809.01620).
+It is a part of the [Chainspace platform](https://chainspace.io/docs/#introduction), but can be used on its own.
+Blockmania is another example of a Total Ordering algorithm.
+
+Typically, clients are external to the network nodes and emit transactions that need to be agreed upon.
+
+Nodes in the network maintain an asymmetric signature scheme and all others can authentically verify their messages via public key.
+Blockmania keep consistent blocks history using a DAG structure, that is subsequently interpreted by each node separately.
+
+The basic inter-node network operation in Blockmania consists of each node periodically broadcasting a block to all other nodes.
+The nodes are loosely synchronized, using a [byzantine clock synchronization](https://lamport.azurewebsites.net/pubs/clocks2.pdf) and emit blocks at regular intervals.
+Blocks from each node form a sequence and each refers to the hash of the previous one, forming a hash chain.
+A block contains information about its creator node, its sequence number in the chain, a sequence of entries representing the content of the block.
+Each block emitted by a node is first a _candidate_ block and contains a position defined by its creator node, its sequence number and its content.
+Each entry in the block content is either a transaction received from a client or a reference to the hash of a valid block received from another node.
+The full block is signed using the private key of its creator and broadcast to all other nodes.
+
+Besides broadcasting blocks, nodes listen to requests from other nodes for specific blocks, and respond by sending them the full contents of the blocks requested.
+
+An honest node includes all transactions received from clients as entries into a block it emits.
+Also, all honest nodes receive and checks for validity all blocks they directly or indirectly reference in their created blocks.
+This means that an honest node receives and stores locally a copy of the full DAG record, starting from the genesis vertex until the last block it emits.
+And if an honest node emits a block, all honest nodes will eventually receive and consider valid (after validating) this block as well as all blocks that it references directly or indirectly in its content entries.
```

Comment:
```suggestion
And if an honest node emits a block, all honest nodes eventually receive and consider it valid (after validating) as well as all blocks that it references directly or indirectly in its content entries.
```

Reply:
Fixed https://github.com/optakt/optakt.github.io/pull/25/commits/e51539edef429ee0583f05806acf8610dd0971a0#diff-af4a1f964639da79798e8b30e7b99b5fb3f29ea0d4da47acd88f0d9c9867be13R450
Missed the future tense in that place, thanks!